src/runtime/lpython/lpython.py

```diff
@@ -13,15 +13,31 @@
 
 # data-types
 
+type_to_convert_func = {
+    "i8": int,
+    "i16": int,
+    "i32": int,
+    "i64": int,
+    "f32": float,
+    "f64": float,
+    "c32": complex,
+    "c64": complex,
+    "c_ptr": lambda x: x,
+    "Const": lambda x: x,
+    "Callable": lambda x: x,
+    "Pointer": lambda x: x,
+}
```

Comment:
For `cpython`, the types `f32/64`, `c32/64` would also support auto-conversions. (We did not update the integration_tests yet for these. I guess we can update the tests later progressively. I think we could experience the auto-conversion with `i32/64` for some time and based on it update the integration_tests for `f32/64`, `c32/64`.

Reply:
What is an auto-conversion?

Reply:
Sorry Sir, probably it should just be `conversion`. Previously, `i32/64` did not do conversion (and we have to explicitly use `int()`) and in this PR they support (implicit/auto) conversion to `integers`.